testapp/src/main/res/layout/window_first_time_miniapp.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent">
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_alignParentStart="true"
+    android:orientation="vertical">
+
+    <RelativeLayout
+      android:id="@+id/firstTimeTopView"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:padding="@dimen/small_8"
+      android:background="@color/window_top_bar">
+
+      <ImageView
+        android:id="@+id/firstTimeCloseWindow"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:padding="@dimen/small_8"
+        android:src="@drawable/ic_close_24px" />
+
+      <TextView
+        android:id="@+id/firstTimeTutorial"
+        android:textStyle="bold"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentStart="true"
+        android:padding="@dimen/small_8"
+        android:text="@string/miniapp_first_time_tutorial"
+        android:textSize="@dimen/text_large_16" />
+
+    </RelativeLayout>
+
+    <ImageView
+      android:id="@+id/firstTimeAppIcon"
+      android:layout_width="@dimen/miniapp_icon_size"
```

Comment:
Maybe we could display the icon a little bigger on this screen. It looks a little too small when it's centered by itself. What do you think?

Reply:
Yeah, I have changed to `@dimen/miniapp_logo_size` = `86dp`.